packages/config/config/devices/0x000c/hs-wx300.json

```diff
@@ -0,0 +1,633 @@
+{
+	"revision": 1,
```

Comment:
```suggestion
```
Where did you get this from? The import script?

Reply:
Yes, that was from the import script

Reply:
Ok we need to remove it then. That was something we tried a while back.